app/controllers/v0/higher_level_reviews_controller.rb

```diff
@@ -4,12 +4,32 @@ module V0
   class HigherLevelReviewsController < AppealsBaseController
     def show
       render json: decision_review_service.get_higher_level_review(params[:id]).body
+    rescue => e
+      log_exception_to_personal_information_log(
+        e,
+        error_class: "#{self.class.name}#show exception",
+        data: { request_data: { id: params[:id] } }
+      )
+      raise
     end
 
     def create
       render json: decision_review_service
         .create_higher_level_review(request_body: request_body_hash, user: current_user)
         .body
+    rescue => e
+      request_data = begin
+                       { body: request_body_hash }
+                     rescue
+                       request_body_debug_data
+                     end
```

Comment:
the intent of the rescue is to capture any exception thrown by `create_higher_level_review`, but there's an off-chance that an exception is thrown before `create_higher_level_review` is called via `request_body_hash`